app/assets/javascripts/components/todo_list_container.js.jsx

```diff
@@ -7,19 +7,35 @@ Chk.TodoListContainer = (function () {
       todos: React.PropTypes.array.isRequired
     },
 
+    filterList: function(event){
+      if (event.target.checked){
+        var updatedList= this.props.todos 
+        _.filter(updatedList, function(todo){return todo.complete_date})
+
+      this.setState({todos:updatedList});
+      }
```

Comment:
I just noticed we are missing an else condition:
```js
if (event.target.checked){
  // blah
} else {
  this.setState({ todos: this.props.todos });
}
```

We need this to undo our filter if someone checks the checkbox again.